### PR TITLE
Add Transnomino.app v4.0

### PR DESCRIPTION
**Pull request by raistlin2912**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/transnomino.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'transnomino' do
+  version '4.0'
+  sha256 'f19444fec6076fa4165d79a4c931ee023651502426df4beb46ff5994c6958426'
+
+  url 'https://transnomino.bastiaanverreijt.com/download/Transnomino-4.0.dmg'
```
Comment (miccal) [inline Casks/transnomino.rb:5]: Please change this to
```
url "https://transnomino.bastiaanverreijt.com/download/Transnomino-#{version}.dmg"
```

Comment (raistlin2912) [reply Casks/transnomino.rb:5]: @miccal OMG! It's true! Sorry!

Comment (miccal) [reply Casks/transnomino.rb]: No problem @raistlin2912

Comment (raistlin2912) [reply Casks/transnomino.rb]: Done

Comment (raistlin2912) [reply Casks/transnomino.rb]: @miccal Hello, there are any problem with PR?

Thanks.

Comment (miccal) [reply Casks/transnomino.rb]: Looks fine to me @raistlin2912, thank you.

**Pull request merged**